core-services/prow/02_config/_config.yaml

```diff
@@ -2297,6 +2296,17 @@ tide:
     - shiftstack/shiftstack-ci
     - shiftstack/bugwatcher
     - shiftstack/merge-bot
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/cluster-api-provider-powervs
```

Comment:
No need to create a separate query, just add your repo to the one directly above. Only difference is that it denies one additional label but that's not a problem.

Reply:
done, ptal at the latest commit.